manuscript/recipes/duplicati.md

```diff
@@ -0,0 +1,115 @@
+hero: Duplicati - Yet another option to backup your exciting stuff. It's good to have options.
+
+# Duplicati
+
+Intro
+
+![duplicati Screenshot](../images/duplicati.jpg)
+
+
+[Duplicati](https://www.duplicati.com/) is a free and open-source backup software to store encrypted backups online For Windows, macOS and Linux (our favorite, yay!).
+
+Similar to the other backup solution options in the Cookbook, you can use Duplicati to backup all our data-at-rest to a wide variety of locations, including, but not limited to:
+
+- FTP servers
+- SSH servers
+- WebDAV endpoints
+- Backblaze B2
+- Tardigrade
+- Microsoft OneDrive
+- Amazon S3
+- Google Drive
+- box.com
+- Mega
+- hubiC
+- many others
+
+!!! note
+    Since Duplicati itself offers no user authentication, this design secures Duplicati behind a [Traefik Forward Auth proxy](/ha-docker-swarm/traefik-forward-auth), so that in order to gain access to the Duplicati UI at all, oauth2 authentication (_to GitHub, GitLab, Google, etc_) must have already occurred.
+
+## Ingredients
+
+1. [X] [Docker swarm cluster](/ha-docker-swarm/design/) with [persistent shared storage](/ha-docker-swarm/shared-storage-ceph.md)
+2. [X] [Traefik](/ha-docker-swarm/traefik_public) and [Traefik-Forward-Auth](/ha-docker-swarm/traefik-forward-auth) configured per design
+3. Credentials for one of the Duplicati's supported upload destinations
+
+## Preparation
+
+### Setup data locations
+
+We'll need a folder to store a docker-compose .yml file, and an associated .env file. If you're following my filesystem layout, create `/var/data/config/duplicati` (*for the config*), and `/var/data/duplicati` (*for the metadata*) as follows:
+
+```
+mkdir /var/data/config/duplicati
+mkdir /var/data/duplicati
+cd /var/data/config/duplicati
+```
+
+### Prepare environment
+
+1. Generate a random passphrase to use to encrypt your data. **Save this somewhere safe**, without it you won't be able to restore!
+2. Seriously, **save**. **it**. **somewhere**. **safe**.
+3. Create duplicati.env, and populate with the following variables (with your appropriate time zone)
+```
+PUID=0
+PGID=0
+TZ=Europe/London
```

Comment:
Is there a list of valid TZ values we can refer the user to? I.e., presumably `New Zealand` wouldn't work, but `Pacific/Auckland` would...

Reply:
Added a link to the Wikipedia page listing them.